jobs/ci-kubernetes-e2e-gce-scalability-canary.env

```diff
@@ -0,0 +1 @@
+# place-holder
```

Comment:
Why do you need these place-holder files? Is this checked in some unit tests?
If so, I'd prefer not checking that because multiple jobs could potentially use the same env file.